package.json

```diff
@@ -113,6 +113,7 @@
     "@types/chrome": "^0.0.154",
     "@types/configstore": "^4.0.0",
     "@types/cpy": "^5.1.0",
+    "@types/debug": "^4.1.7",
```

Comment:
Was getting a type error for missing this because puppeteer isn't expecting us to reach into some of their internals:

https://github.com/GoogleChrome/lighthouse/blob/4459f497c7108b94808f79c0cf6cc3bb5613844d/clients/lightrider/lightrider-entry.js#L12-L13